handout/features/typings.md

```diff
@@ -3,9 +3,7 @@
 Astute readers might be wondering what happens when TypeScript programmers need
 to interface with JavaScript modules that have no type information.  TypeScript
 recognizes files labelled `*.d.ts` as _definition_ files.  These files are
-meant to use TypeScript to describe interfaces presented by JavaScript 
+meant to use TypeScript to describe interfaces presented by JavaScript
 libraries.
 
-There are communities of people dedicated to creating typings for JavaScript
```

Comment:
@bennett000 Yes, `typings` may still be needed in 1.x versions as they do not support `@types` very well. Please see https://github.com/rangle/ngCourse2/pull/771/commits/df51441f0687639f653feb747d5cd6f018a22b66.